src/lib/webengine/webview.cpp

```diff
@@ -392,14 +392,16 @@ void WebView::printPage()
 {
     Q_ASSERT(m_page);
 
-    QPrintDialog* dialog = new QPrintDialog(this);
+    QPrinter* printer = new QPrinter(QPrinter::HighResolution);
```

Comment:
Any reason for `QPrinter::HighResolution` instead of default?

Reply:
No specific reason, reverted it back to default.